example-config.yml

```diff
@@ -668,4 +668,7 @@ itinerary:
 ### Setting this to true will use the route viewer sort algorithm on routes
 #   useRouteViewSort: false
 ### Setting this to false will hide the shadowey dot that appears when dragging the map
-#   showShadowDotOnMapDrag: false
\ No newline at end of file
+#   showShadowDotOnMapDrag: false
+### In routes, this will always show the routeLongName before the headsign
+### !!! WARNING: in some feeds, this may show the long name twice. In some feeds, this may crash !!!
```

Comment:
😬😬😬 Adding a config item that could cause a crash feels wrong, especially since `route_long_name` is only conditionally required

Reply:
Yeah it's not a feature I'm crazy about, which is why I'm trying to make sure that only people who really know what they're doing use it